flutter/lib/resources/result_manager.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:convert';
+import 'dart:io';
+
+class ResultManager {
+  static const _jsonResultFileName = 'result.json';
```

Comment:
Do you have the file `result.json` in the app root folder on Android after running benchmark? I have one on iOS but not on Android.

Reply:
Yes, I see this file on my Android phone.